### PR TITLE
fix(plugins): use compiled dependency instead of `css-expand-shorthand`

### LINKED ISSUE #1534: [IE11] Dependency js file has arrow function
## Bug Report
Stardust uses `css-shorthand-expand` which contains a js file with an arrow function.
This breaks IE11 compatibility:

See line 28:
https://github.com/kapetan/css-shorthand-expand/blob/master/source/background.js

### Steps
Open https://stardust-ui.github.io/react/ in IE11.

### Expected Result
Page renders something.

### Actual Result
Page doesn't render anything.

### Version
0.33.0

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #1534.

### Problem

IE11 does not have support of arrow functions while our dependency [`css-shorthand-expand`](https://github.com/kapetan/css-shorthand-expand) has its [usage](https://github.com/kapetan/css-shorthand-expand/blob/master/source/background.js#L28). And for client there is only one option to fix - compile these dependency with `babel-loader` or any other thing 👎 

### Solution

I used [`@pika/pack`](https://github.com/pikapkg/pack) to quicky get bundled library, so our `cssExpandShorthand.ts` contains `css-shorthand-expand` and all its dependencies in the single and tree-shaken file. As we compile all our files with Babel: arrow functions and any other unsupported things will be compiled properly 🚀

### CODE REVIEW

File: docs/src/index.ejs [hunk 1]
```diff
@@ -32,24 +31,14 @@
   </style>
 </head>
 <body>
-  <script src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7/dist/polyfill.min.js"></script>
+  <script src='https://cdn.jsdelivr.net/npm/core-js-bundle/minified.js'></script>
+  <script src='https://cdn.jsdelivr.net/npm/whatwg-fetch/dist/fetch.umd.min.js'></script>
```
Comment (layershifter) [inline docs/src/index.ejs:35]: `fetch()` is not supported by IE11

File: docs/src/index.ejs [hunk 2]
```diff
@@ -32,24 +31,14 @@
   </style>
 </head>
 <body>
-  <script src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7/dist/polyfill.min.js"></script>
+  <script src='https://cdn.jsdelivr.net/npm/core-js-bundle/minified.js'></script>
```
Comment (layershifter) [inline docs/src/index.ejs:34]: `core-js-bundle` is bundled version of `core-js` which is a successor for `@babel/polyfill`:
https://babeljs.io/docs/en/babel-polyfill

File: docs/src/index.ejs [hunk 3]
```diff
@@ -32,24 +31,14 @@
...
-    src="https://unpkg.com/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-babylon.js"
-  ></script>
-  <script
-    crossOrigin="anonymous"
-    src="https://unpkg.com/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-html.js"
-  ></script>
-  <script
-    crossOrigin="anonymous"
-    src="https://unpkg.com/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-typescript.js"
+    src="https://cdn.jsdelivr.net/combine/npm/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/standalone.min.js,npm/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-babylon.min.js,npm/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-html.min.js,npm/prettier@<%= htmlWebpackPlugin.options.versions.prettier %>/parser-typescript.min.js"
```
Comment (layershifter) [inline docs/src/index.ejs:39]: Instead of 4 files unminified files from CDN we will require only minified one 🚤

File: docs/src/index.ejs [hunk 4]
```diff
@@ -62,7 +51,8 @@
 
   <script src="https://cdn.jsdelivr.net/npm/@babel/standalone@<%= htmlWebpackPlugin.options.versions.babelStandalone %>/babel.min.js"></script>
 
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.16.0/themes/prism-tomorrow.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.16.0/themes/prism-tomorrow.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/react-vis@<%= htmlWebpackPlugin.options.versions.reactVis %>/dist/style.min.css" />
```
Comment (layershifter) [inline docs/src/index.ejs:55]: Also minified versions

File: packages/docs-components/src/CodeSnippet/CodeSnippetLabel.tsx [hunk 5]
```diff
@@ -28,14 +43,14 @@ const CodeSnippetLabel: React.FunctionComponent<CopySnippetLabelProps> = props =
...
-          top: '1rem',
+          right: '0.8rem',
+          top: '0.8rem',
           zIndex: 100,
         }}
         title={copyable ? 'Copy' : undefined}
       >
         <div>{label || mode}</div>
-        {copyable && <div style={{ marginLeft: '5px' }}>{active ? '✔ ' : '⎘'}</div>}
+        {copyable && <div style={{ marginLeft: '5px' }}>{active ? checkIcon : copyIcon}</div>}
```
Comment (layershifter) [inline packages/docs-components/src/CodeSnippet/CodeSnippetLabel.tsx:53]: image

Current icon looks ugly and there are no good replacement, so I decided to use SVGs.
Also it's not enough contrast, so change `color` to `#ccc` from PrismJS theme.


### After

codesnippet-icons

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**